tests/load-tests/load-tests-example.json

```
@@ -15,10 +15,13 @@
   "createResourcesTimeMax": 12.397832976,
   "runPipelineSucceededTimeAvg": 269.6,
   "runPipelineSucceededTimeMax": 471,
-  "runPipelineFailedTimeAvg": 0,
+  "runPipelineFailedTimeAvg": 121.456,
   "deploymentSucceededTimeAvg": 4.733333333333333,
   "deploymentSucceededTimeMax": 7,
   "deploymentFailedTimeAvg": 0,
+  "its_runPipelineSucceededTimeAvg": 14.938775510204081,
+  "its_runPipelineSucceededTimeMax": 85,
+  "its_runpipelineFailedTimeAvg": 0,
```

@naftalysh Please keep the naming consistent
```suggestion
  "itsRunPipelineSucceededTimeAvg": 14.938775510204081,
  "itsRunPipelineSucceededTimeMax": 85,
  "itsRunPipelineFailedTimeAvg": 0,
```

---

Additionally, `itsRunPipelineFailures` and `itsRunPipelineFailureRate` is missing from this example

---

Updated